docs/manual/introduction/Browser-support.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<base href="../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+<body>
+	<h1>[name]</h1>
+
+	<h2>Overview</h2>
+	<div>
+		<p>
+			<strong>TL;DR</strong> three.js supports most modern browsers, including Internet Explorer 11 and above.</p>
+		<p>
+			three.js provides WebGL, Canvas, SVG and CSS renderers. Different renderers may have different requirements of compatibility.
```

Comment:
Three.js at the start of a sentence should be capitalised.

Reply:
I disagree, and always write it as `three.js`, even at the start of a sentence. See the [wikipedia article for eBay](https://en.wikipedia.org/wiki/EBay), for example -- or iPad.

If that is bothersome to you, you can rewrite the sentence and avoid `three.js` as the first word.

Reply:
We did discuss this before and the consensus then was to capitalize it at the start of sentences.

Reply:
@looeee Where was this discussed?

Reply:
If `three.js` is correct, the corresponding [wikipedia article](https://en.wikipedia.org/wiki/Three.js) needs a revision 😁 

BTW: I always write it as `three.js`, too.

> We did discuss this before and the consensus then was to capitalize it at the start of sentences.

Can't remember. A link to this discussion would be helpful...

Reply:
I personally don't mind. `Three.js` or `three.js` are fine. I have more of a problem with `ThreeJS`, `threeJS`, `THREEjs` and other variations...